Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -591,7 +591,7 @@ - (void)waitForDownloadsForUser:(RLMUser *)user
     NSAssert(session, @"Cannot call with invalid partition value");
     XCTestExpectation *ex = expectation ?: [self expectationWithDescription:@"Wait for download completion"];
     __block NSError *theError = nil;
-    BOOL queued = [session waitForDownloadCompletionOnQueue:nil callback:^(NSError *err) {
+    BOOL queued = [session waitForDownloadCompletionOnQueue:dispatch_get_global_queue(0, 0) callback:^(NSError *err) {
```

Comment:
These are because running the runloop (via `waitForExpectations`) in an async context on the main thread seems to not dispatch the main queue, so these need to be delivered to a different queue.